bin/git-theta-filter

```diff
@@ -57,12 +76,9 @@ def clean(args):
     staged_file_contents = OrderedDict({"model_dir": theta_model_dir})
     # Sort the keys so we don't get changing diffs based on serialization order.
     for keys, param in sorted(utils.flatten(model_checkpoint).items()):
-        # TODO(bdlester): We should update this staged_file_content from having
-        # different keys for shape, dtype, and hash to a dict.
-        param_name = "/".join(keys)
-        staged_file_contents[f"{param_name} shape"] = params.get_shape_str(param)
-        staged_file_contents[f"{param_name} dtype"] = params.get_dtype_str(param)
-        staged_file_contents[f"{param_name} hash"] = params.get_hash(param)
+         param_name = "/".join(keys)
```

Comment:
It looks like you have 5 spaces here instead of 4.

We need to get black running (see https://github.com/r-three/git-theta/issues/88) but until then can you fix this formatting?

Reply:
Thanks for the catch. I changed it